### PR TITLE
Fix install for MacOSX (trilinos/Trilinos#7881)

### PR DESCRIPTION
**Pull request by bartlettroscoe**
Should fix install on MacOSX systems (see trilinos/Trilinos#7881)

See the commits for more details

## How was this tested?

I only run the updated unit tests.  I don't have access to a Mac right now to test this myself.

### CODE REVIEW

**Comment by bartlettroscoe** [comment]
CC: @ndellingwood

@jmgate, do you have time to review this TriBITS PR before I merge and then snapshot to a Trilinos branch to merge to Trilinos 'develop'? 

If we are going to be doing a real review, the that should occur in a PR in the TriBITS repo and not in snapshots to the Trilinos repo (that strips out all of the TriBITS tests).

**Comment by jmgate** [comment]
Sorry I missed this, @bartlettroscoe.  Apparently I need to adjust my GitHub notification settings.

**Comment by jmgate** [comment]
Unfortunately I won't have time to review this until next week.

**Comment by bartlettroscoe** [comment]
> Unfortunately I won't have time to review this until next week, but I have it open now, so I'll get to it early on Monday.

@jmgate, that is okay.  GitHub allows you to review a PR even after it has been merged.  Then I can address any issues post-merge in a follow-up PR.

**Review by jmgate — COMMENTED** [review]
This looks good to me, @bartlettroscoe.  Thanks.

**Review by e10harvey — COMMENTED** [review]
In general, everything looks great. Please see suggestions. Thanks, Ross!

File: tribits/core/package_arch/TribitsAddInstallGroupAndPermsFixups.cmake [hunk 1]
```diff
@@ -2,6 +2,29 @@ INCLUDE(Join)
 INCLUDE(TribitsFilepathHelpers)
 
 
+FUNCTION(TRIBITS_DETERMINE_IF_SETUP_FOR_GROUP_AND_PERMS_MODIFICATIONS
+  SETUP_FOR_GROUP_AND_PERMS_MODIFICATIONS_OUT
+  )
+
+  IF (CMAKE_SYSTEM_NAME STREQUAL "Windows")
+    SET(setupForGroupAndPermsModifications FALSE)
+  ELSEIF(
```
[the remark this answers is not in the document]
Comment (bartlettroscoe) [reply tribits/core/package_arch/TribitsAddInstallGroupAndPermsFixups.cmake:11]: This should be addressed in the new PR #331.

**Pull request merged**